scripts/deploy/containerregistry.sh

```diff
@@ -82,6 +83,18 @@ fi
 
 sleep 10
 
+if ( echo "${DESTINATION_REGISTRY_HOST}" |grep -q icr.io ) && [ "$CISO_CODESIGN_ENABLE" == "true" ]; then 
```

Comment:
@Amy0511 Maybe just syntax consistency & readability move the `grep` command to a line before, e.g.
```
SUPPORTED_SIGNED_REGISTRY=$(echo "${DESTINATION_REGISTRY_HOST}" | grep icr.io)
if [ -z "$SUPPORTED_SIGNED_REGISTRY" ] && [ "$CISO_CODESIGN_ENABLE" == "true" ]; then
```